lang/node/Makefile

```diff
@@ -60,13 +71,37 @@ CONFIGURE_ARGS= \
 ifneq ($(findstring arm,$(ARCH)),)
 CONFIGURE_ARGS+= \
 	$(if $(CONFIG_SOFT_FLOAT),--with-arm-float-abi=soft,--with-arm-float-abi=hard)
+
+ifneq ($(findstring vfp,$(CPU_SUBTYPE)),)
+CONFIGURE_ARGS+= --with-arm-fpu=vfp
+endif
+
+ifneq ($(findstring vfpv3,$(CPU_SUBTYPE)),)
+CONFIGURE_ARGS+= --with-arm-fpu=vfpv3
+endif
+
+ifneq ($(findstring vfpv3-d16,$(CPU_SUBTYPE)),)
+CONFIGURE_ARGS+= --with-arm-fpu=vfpv3-d16
+endif
+
+ifneq ($(findstring neon,$(CPU_SUBTYPE)),)
+CONFIGURE_ARGS+= --with-arm-fpu=neon
+endif
+
```

Comment:
1. Sorry I made mistake. it was 'CONFIG_CPU_TYPE', not 'CPU_SUBTYPE.' 

2. To prevent matching to unexpected strings, only use 'neon' and 'vfpv3'
('vfp' is default value)